### ISSUE TITLE
[LINT_BUG]: Unused universally attached package functions not linting

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#79]
### box.linters version

0.0.0.9013

### Sample source code to lint
```R
box::use(
  path/to/stringr[...]
)
```
### Lint command used
```R
lintr::lint("code.R", linters = box_unused_attached_pkg_linter())
```
### Lint result

No lint!

### Expected result

Should lint on unused functions attached by `...`

**Comment by radbasa** [comment]
```R
box::use(
  path/to/stringr[...]
)
```

`path/to/stringr[...]` is for modules. This is a package. Proper call should be:

```R
box::use(
  stringr[...]
)
```
